CONTRIBUTING.md

```diff
@@ -1,14 +1,44 @@
-Contributing to the AWS SDK for Go
+# Contributing to the AWS SDK for Go
 
+Thank you for your interest in contributing to the AWS SDK for Go!
 We work hard to provide a high-quality and useful SDK, and we greatly value
 feedback and contributions from our community. Whether it's a bug report,
 new feature, correction, or additional documentation, we welcome your issues
 and pull requests. Please read through this document before submitting any
-issues or pull requests to ensure we have all the necessary information to
+[issues][] or [pull requests][pr] to ensure we have all the necessary information to
```

Comment:
Do these markdown links need to be filled in? looks like the `issues` one does?  I think you can also use just `[issues]` without the trailing `[]` to like to a reference of the same name.

Reply:
Link works as is but suggested formatting is cleaner. Fixed!